src/test/java/de/tum/in/www1/artemis/authentication/PersonalAccessTokensTest.java

```diff
@@ -0,0 +1,73 @@
+package de.tum.in.www1.artemis.authentication;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.test.context.support.WithAnonymousUser;
+import org.springframework.security.test.context.support.WithMockUser;
+
+import de.tum.in.www1.artemis.AbstractSpringIntegrationJenkinsGitlabTest;
+import de.tum.in.www1.artemis.domain.User;
+import de.tum.in.www1.artemis.repository.UserRepository;
+
+class PersonalAccessTokensTest extends AbstractSpringIntegrationJenkinsGitlabTest {
+
+    private static final String TOKEN_API_URL = "/api/personal-access-token";
+
+    private static final String ACTIVATE_NAME = "student1";
+
+    private static final String NOT_ACTIVATE_NAME = "student2";
+
+    @Value("${artemis.personal-access-token.max-lifetime-in-days}")
+    private long personalAccessTokenMaxLifetimeDays;
+
+    @Autowired
+    private UserRepository userRepository;
+
+    @BeforeEach
+    public void setUp() {
+        database.addUsers(2, 0, 0, 0);
+
+        User notActivated = userRepository.findOneWithGroupsAndAuthoritiesByLogin(NOT_ACTIVATE_NAME).get();
+        notActivated.setActivated(false);
+        userRepository.save(notActivated);
+    }
```

Comment:
Test for the GET method is missing. If you replace is as suggested, it's not needed, though.